data/kernels_and_extensions.yml

```diff
@@ -0,0 +1,30 @@
+Links:
+  Python: https://github.com/python/python
+  Matlab: https://github.com/mathworks/jupyter-matlab-proxy
+  R: https://github.com/IRkernel/IRkernel
+  Octave: https://github.com/Calysto/octave_kernel
+  Sage: https://github.com/sagemath/sage
+  gnuplot: https://github.com/has2k1/gnuplot_kernel
+  Mathematica: https://github.com/WolframResearch/WolframLanguageForJupyter
+  Cling: https://github.com/jupyter-xeus/xeus-cling
+  evcxr: https://github.com/evcxr/evcxr
+  Gophernotes: https://github.com/gopherdata/gophernotes
+  Calysto Scheme: https://github.com/Calysto/calysto_scheme
```

Comment:
Why is the prefix here 'calysto'? This is not the programming language, right? It is the person you created the kernel. Should that be in the 'key'?
However, we have to think of resolving ambiguities where several kernels exist for one programming language. Can you come up with an idea?